### DISCUSSION TITLE
How to maintain folder & file icons in your Gruvbox theme?

### DISCUSSION THREAD

**Discussion by kenanmike** [BryanJenksCommunity/FAQ#105] [Q&A] [answered]
I love your being able to use your excellent theme and I'm very grateful to you for making it publicly available. How do I maintain the original icons for folders and files in file explorer? Or how can I add in some nice ones rather than having empty boxes there (see screenshot)?
![image](https://user-images.githubusercontent.com/85374001/129860002-00463139-8a6c-4580-b26e-e13c8bf89fdc.png)

**Comment by tallguyjenks** [comment, marked as answer]
Empty boxes are due to a missing font for default icons see section `2.2.2`:

![image](https://user-images.githubusercontent.com/29872822/129986972-de6e7f6f-7b41-4aeb-85c4-f38ec03ddaf5.png)


to customize your specific folders see section `2.2.3`:

![image](https://user-images.githubusercontent.com/29872822/129987012-9ae3de37-bff5-4c53-a58b-3fc1f7fd9b64.png)
